tycho-its/src/test/java/org/eclipse/tycho/test/versionsplugin/TychoVersionsPluginTest.java

```diff
@@ -79,4 +80,115 @@ public void updateTargetVersionTest() throws Exception {
 						&& targetContent.contains("sequenceNumber=\"12\""));
 	}
 
+	/**
+	 * Verifies that the update-pom goal of the tycho-version plug-in updates the
+	 * version of a pom when the pom file is implicit. The command line for this
+	 * would be
+	 * <p>
+	 * <code>mvn org.eclipse.tycho:tycho-versions-plugin:update-pom</code>
+	 * </p>
+	 * This was created in response to
+	 * <a href="https://github.com/eclipse-tycho/tycho/issues/309">issue 309</a>.
+	 */
+	@Test
+	public void testUpdatePomWithImplicitPomName() throws Exception {
+		String POM_NAME = "pom.xml";
+		String MANIFEST_VERSION = "2.0.0";
+
+		Verifier verifier = getVerifier("tycho-version.update-pom.pomNamedPomXml", false);
```

Comment:
Can you please organize the projects in subfolders like this?

```suggestion
		Verifier verifier = getVerifier("tycho-version-plugin/update-pom/pomNamedPomXml", false);
```